packages/core/src/integrations/inboundfilters.ts

```diff
@@ -5,6 +5,16 @@ import { getEventDescription, logger, stringMatchesSomePattern } from '@sentry/u
 // this is the result of a script being pulled in from an external domain and CORS.
 const DEFAULT_IGNORE_ERRORS = [/^Script error\.?$/, /^Javascript error: Script error\.? on line 0$/];
 
+const DEFAULT_IGNORE_TRANSACTIONS = [
```

Comment:
fwiw, I've also seen ping

Reply:
we decided to keep consistency with https://docs.sentry.io/product/performance/performance-at-scale/#health-checks

but we can also change it there

Reply:
had a customer email me today that `ping` was 13 TPM while the rest were all .01 TPM and below. said it ate up their quota.

Reply:
it was python